first.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+// 1 ---------------
+
+const numberSetOne = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+function modifyNumbers(numbers) {
+    const modifyNumberSet = [];
+
+    for (let i = 0; i < numbers.length; i++) {
+        // if (numbers[i] % 2 === 0) {
+        //     modifyNumberSet.push(numbers[i] += 1);
+        // } else {
+        //     modifyNumberSet.push(numbers[i] -= 1);
+        // }
+        let currentNumber = numbers[i];
```

Comment:
why `let`?

Reply:
this is a changeable variable, and it doesn't work with `const`